crates/sozo/src/commands/events.rs

```diff
@@ -0,0 +1,51 @@
+use crate::ops::events;
+
+use super::options::{
+    dojo_metadata_from_workspace, starknet::StarknetOptions, world::WorldOptions,
+};
+use anyhow::Result;
+use clap::Args;
+use scarb::core::Config;
+
+#[derive(Args, Debug)]
+pub struct EventsArgs {
+    #[clap(short, long)]
+    #[clap(help = "Number of events to return per page")]
+    pub chunk_size: u64,
+
+    #[clap(short, long)]
+    #[clap(help = "Block number from where to look for events")]
+    pub from_block: Option<u64>,
+
+    #[clap(short, long)]
+    #[clap(help = "Block number until where to look for events")]
+    pub to_block: Option<u64>,
+
+    #[clap(short, long)]
+    #[clap(help = "List of specific events to be filtered")]
```

Comment:
I think we should specify the `value_delimiter` to be `,` here. Doesn't seem like it's parsing `ComponentRegistered,PlayerJoined` as 2 values. It also better if we can put in the help message how the value should be passed.

Also maybe it's worth adding a unit test for this ( we should be doing it for other commands also tbh ). Just a simple test to make sure the args are parsed properly.

Reply:
what would be a good way to test this, like pass in a string to `EventsArgs` and it gives a formed struct back? I couldn't find this in docs.

Reply:
I think it should be safe to replace `Args` with [`Parser`](https://docs.rs/clap/latest/clap/trait.Parser.html) attr. then you can use  [`Parser::parse_from`](https://docs.rs/clap/latest/clap/trait.Parser.html#method.parse_from) like so:

```rust
let args = EventsArgs::parse_from(&["events", "Event1,Event2", "--chunk-size", "1"]);

assert_eq!(args.events.unwrap().len(),2);
```